content/building/encrypting.md

```diff
@@ -29,7 +29,7 @@ On macOS, in order to base64 encode a file and copy the contents to your clipboa
   ```bash
   cat your_file_name.extension | base64 | pbcopy 
   ```
-After the command has been run, you can paste the string from your clipboard to our encryption tool and then save the encrypted string to an environment variable.
+After the command has been run, you can paste the string from your clipboard to our encryption tool in the Workflow Editor and then save the encrypted string to an environment variable.
```

Comment:
It's not really the workflow editor as the same interface is available in both yaml view and in the app settings for Flutter apps, and workflow editor strictly refers to the UI settings of Flutter apps. Perhaps use "in the Codemagic UI" instead?

Reply:
Agreed, Codemagic UI sounds a lot better, going to submit it